main.tf

```diff
@@ -207,12 +207,25 @@ resource aws_vpn_gateway_route_propagation "vpn_routes_private" {
 # Flow Logs
 ###########
 
+resource aws_flow_log "vpc_log" {
+  count                = "${var.build_s3_flow_logs ? 1 : 0}"
+  log_destination      = "${aws_s3_bucket.vpc_log_bucket.arn}"
+  log_destination_type = "s3"
+  vpc_id               = "${aws_vpc.vpc.id}"
+  traffic_type         = "ALL"
+}
+
+resource "aws_s3_bucket" "vpc_log_bucket" {
+  count = "${var.build_s3_flow_logs ? 1 : 0}"
+  tags  = "${merge(local.base_tags, map("Name", format("%s-VPNGateway", var.vpc_name), "transitvpc:spoke", var.spoke_vpc), var.custom_tags)}"
+}
+
 resource aws_flow_log "main" {
-  count          = "${var.build_flow_logs ? 1 : 0}"
```

Comment:
See earlier comment regarding multiple flow log resources.  If both are needed, we should update the name for this resource to denote CWLogs in some way, as `main` is ambiguous.

Reply:
ambiguous name is updated.